src/test/java/org/hiero/sketch/MembershipTest.java

```diff
@@ -25,6 +26,19 @@ public void TestFullMembership() {
         assert(FM.isMember(7));
         assert(!FM.isMember(-2));
         assert(!FM.isMember(-10));
+        IMembershipSet mysample = FM.sample(5);
+        System.out.println("printing sample of 5 elements: ");
+        PrintMemebership(mysample);
+
+    }
+
+    private void PrintMemebership(IMembershipSet myset) {
```

Comment:
The standard Java way is to have a toString() method. The benefit of this is that the debugger calls it for you automatically when you want to see the value. However, for large structures toString() should be done carefully: 
- use a StringBuilder, because string concatentation is linear time, so concatenating n strings turns into quadratic time
- perhaps truncate the result to a prefix if the data structure is too large (print an ellipsis)